sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -1,477 +1,225 @@
-WITH _metrics_ping_distinct_client_count AS (
-  SELECT
-    DATE(submission_timestamp) AS submission_date,
-    COUNT(DISTINCT client_info.client_id) AS metrics_ping_distinct_client_count
-  FROM
-    firefox_ios.metrics
+WITH baseline_clients AS (
+  SELECT DISTINCT
```

Comment:
Is DISTINCT enough here, can we be sure that a single client will not ever get a baseline ping with different country values within the same day?